google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseInfo.java

```diff
@@ -44,6 +45,11 @@
      */
     public abstract Builder setEncryptionConfig(CustomerManagedEncryption encryptionConfig);
 
+    /** The read-write region which will be used for the database's leader replicas. */
```

Comment:
Would it be possible (and make sense) to reference some online documentation that indicates what the possible values are, including the format that is expected?

Reply:
I don't think we have published any doc yet. The possible value depends on the returned values in `leader_options`.

Reply:
The format should like: https://cloud.google.com/spanner/docs/instances#available-configurations-multi-region

Reply:
Just confirmed with the backend team. The option would not have the `regional-` prefix. It will be something like `us-east1` and `us-central1`.

Reply:
@thiagotnunes I think the best we can do is point to [the multi-region instance configs page](https://cloud.google.com/spanner/docs/instances#available-configurations-multi-region) and say that all RW region names are valid. And that you can use `getInstanceConfig` to fetch the `leader options` available.

Reply:
Updated the javadoc with the link to the possible values here.